src/common/structures/__init__.py

```diff
@@ -15,7 +15,8 @@
     CORNER_REFINEMENT_METHOD_DICTIONARY_TEXT_TO_INT
 from .detection_parameters import DetectionParameters
 from .detector_resolution import DetectorResolution
-from .detector_status import DetectorStatus
+from .capture_status import CaptureStatus
+from .marker_status import MarkerStatus
```

Comment:
Minor detail - I've tried to keep the imports in alphabetical order where possible. It's not worth going out of the way to fix now, but if you modify an __init__ file and notice it, please feel free to address it.